src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -81,10 +81,16 @@ class SidebarPanels extends React.Component<Props> {
 
     detailsSidebar: ElementRefType = React.createRef();
 
+    isFirstLoad: boolean = true;
+
     metadataSidebar: ElementRefType = React.createRef();
 
     versionsSidebar: ElementRefType = React.createRef();
 
+    componentDidMount() {
+        this.isFirstLoad = false;
```

Comment:
Seems like this should be tracked in state so changes to it cause a re-render. Also, maybe `isMounted` or `isInitialized`? That way the default prop value might not be necessary in each sidebar?